ansible/roles/nginx-rails/tasks/main.yml

```diff
@@ -9,7 +9,15 @@
   file:
     src: /etc/nginx/sites-available/{{project_name}}
     dest: /etc/nginx/sites-enabled/{{project_name}}
-    state: "{{ (ssl_enabled and nginx_ssl_config is defined) | ternary('absent', 'link') }}"
+    state: link
+  when: ssl_enabled != true or nginx_ssl_config is not defined
+  become: true
```

Comment:
I think the old code removed it if ssl was no longer enabled which this won't do if I'm reading it correctly

Reply:
ok wait actually i think this block and the next block are switched